doc/CONFIGURATION.md

```diff
@@ -358,3 +358,14 @@ We welcome feedback on how this works for your applications and workloads.
 ## Logging
 
 By default, Mountpoint emits high-severity log information to [syslog](https://datatracker.ietf.org/doc/html/rfc5424) if available on your system. You can change what level of information is logged, and to where it is logged. See [LOGGING.md](LOGGING.md) for more details on configuring logging.
+
+## Unstable configurations
+
+Configurations in this section are considered unstable and may be removed at any time. We do not recommend using them unless other options are not available to you.
```

Comment:
I'd be tempted to make this recommendation a bit stronger

```suggestion
## Unstable configurations

Configurations in this section are considered unstable and may be removed at any time. We do not recommend using them unless otherwise advised.
```